07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,70 @@
+ .links{
+  background-color:black;
```

Comment:
Looks like you are some CSS short of completing this assignment. Let me know and we can get together and finish it up.